examples/basic_bot_functions.py

```diff
@@ -0,0 +1,104 @@
+from pyboy.utils import WindowEvent
+import pyboy as pyboy
+import io
+
+'''
+This shows the basic functions you can use if you want to write a bot using only the game image as a human would see it.
+Uses Super Mario Land 1 as an example.
+'''
+
+path_to_rom = '../SuperMarioLand1.gb'
```

Comment:
Have a look in the other examples. It would be better if the user can specify the location